yarn-project/noir-contracts/src/contracts/noir-aztec3/src/abi.nr

```diff
@@ -176,10 +176,11 @@ fn empty_contract_storage_update_request() -> ContractStorageUpdateRequest {
 // + CONTRACT_STORAGE_UPDATE_REQUESTS_LENGTH * CONTRACT_STORAGE_UPDATE_REQUEST_SIZE
 // + CONTRACT_STORAGE_READS_LENGTH * CONTRACT_STORAGE_READ_SIZE
 // + MAX_PUBLIC_CALL_STACK
+// + MAX_NEW_COMMITMENTS
 // + MAX_L1_MSG_STACK
```

Comment:
Looks like I misseed an `L1_MSG_STACK` back when I were renaming long time ago. Used in 
```
new_l2_to_l1_msgs: [Field; crate::abi::MAX_L1_MSG_STACK],
```
Higher up as well.